### PR TITLE
Change TELCODE to CAMCODE for HeaderService

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: etc/ComCam/primary_hdu.header [hunk 1]
```diff
@@ -54,7 +54,7 @@ FOCUSZ  =                      / Focus Z position
 OBJECT  =                      / Name of the observed object
 
         ---- Image-identifying used to build OBS-ID ----
-TELCODE =                      / The code for the telescope (AT/CC/MC)
+CAMCODE =                      / The code for the telescope (AT/CC/MC)
```
Comment (mareuter) [inline etc/ComCam/primary_hdu.header]: Should say "code for the camera"

File: etc/LATISS/primary_hdu.header [hunk 2]
```diff
@@ -76,7 +76,7 @@ EQUINOX =             2000.00  / Equinox of coordinates
 RADESYS =                      / Telescope coordinate system
 
         ---- Image-identifying used to build OBS-ID ----
-TELCODE =                      / The code for the telescope (AT/CC/MC)
+CAMCODE =                      / The code for the telescope (AT/CC/MC)
```
Comment (mareuter) [inline etc/LATISS/primary_hdu.header]: Ditto from above.

**Review by mareuter — APPROVED** [review]
Small comment to address. Otherwise, this looks fine to me.

File: etc/ComCam/primary_hdu.header (hunk 1 above)
Comment (menanteau) [reply etc/ComCam/primary_hdu.header]: Fixed, thanks

File: etc/LATISS/primary_hdu.header (hunk 2 above)
Comment (menanteau) [reply etc/LATISS/primary_hdu.header]: Fixed too.

**Pull request merged**